packages/electrode-archetype-react-app-dev/lib/dev-admin/dev-hapi17.js

```diff
@@ -3,9 +3,10 @@
 /* eslint-disable no-console, no-magic-numbers */
 
 const Url = require("url");
-const Boom = require("boom");
+const Boom = require("@hapi/boom");
```

Comment:
boom is only used at line 51 in this file and no where else, maybe just remove it, and create a simple 404 response.

Reply:
the `replyNotFound` method doesn't seem to be used.  Let's just remove that and boom.